images/executor/README.md

```diff
@@ -0,0 +1,59 @@
+# Executor
+
+This image will be used to execute course code in isolation in production environments.
+
+## Release process
+
+This image is built and pushed to the container registry as `prairielearn/executor:VERSION`, where version is a string in the `EXECUTOR_VERSION` file in the root of the PrairieLearn repository. This allows us to consistently identify which version of this container should be used by any given version of PrairieLearn. For example, if some revision of PrairieLearn has an `EXECUTOR_VERSION` file with the contents `v1.2.3`, that version of PrairieLearn will execute code in the `prairielearn/executor:v1.2.3` image.
```

Comment:
Yeah, I didn't really think to deeply about this when I was first writing it. I agree that semver isn't really necessary here since we're always going to want the executor pinned to a specific version anyways.

Reply:
I ended up going with images tagged with the specific PrairieLearn Git commit they're being built from - I think that resolves this!